src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -121,6 +121,7 @@ form.comment {
     float: right;
     margin-left: 10px;
     margin-right: -5px;
+    max-height: 50px;
```

Comment:
This fixes the bug where the dropdown would get misaligned on successive comments.